tests/integration/values/values_common.yaml

```diff
@@ -0,0 +1,73 @@
+sumologic:
```

Comment:
Can you also add a short comment here, explaining what this is for?

Reply:
Done